test/integration/targets/inventory_vmware_vm_inventory/runme.sh

```diff
@@ -59,7 +65,7 @@ ${PYTHON} -m pip install toml
 ansible-inventory -i ${VMWARE_CONFIG} --list --toml
 
 echo "Check if cache is working for inventory plugin"
-ls "$(pwd)/inventory_cache/vmware_vm_*" > /dev/null 2>&1
+ls $(pwd)/inventory_cache/vmware_vm_* > /dev/null 2>&1
```

Comment:
AFAICT, the final playbook was never being run because of this. The quotes made ls interpret the wildcard literally, failing and dropping to the cleanup code early.